src/hooks/usePromptForParty.ts

```diff
@@ -8,22 +8,24 @@ export function usePromptForParty(): boolean | null {
   const parties = useParties();
 
   if (!profile?.partyId || !parties) {
+    console.log('null');
```

Comment:
Remove the calls to `console.log` (we have automatic checks for this as well, as you can see in the files tab).